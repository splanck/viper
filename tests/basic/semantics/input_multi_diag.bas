10 INPUT A, B
20 PRINT A
30 PRINT B
40 END
