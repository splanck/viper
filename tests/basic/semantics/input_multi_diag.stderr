Trap @main#2 line 1: DomainError (code=0): INPUT: expected 2 values, got 1
